llvm/include/llvm/MC/MCAssembler.h

```diff
@@ -225,6 +228,9 @@ class MCAssembler {
     return make_pointee_range(Symbols);
   }
 
+  void setCPU(std::string TargetCPU) { CPU = std::move(TargetCPU); }
+  StringRef getCPU() const { return CPU; }
+
```

Comment:
Is this used anywhere? The CPU is emitted via the PPCTargetStreamer, so I think there is no use for it here.